src/components/Toasts/TransactionToast.tsx

```diff
@@ -41,15 +41,15 @@ export type TransactionToastProps = {
   /**
    * Token Info of sender token
    */
-  senderToken?: TokenInfo;
+  senderToken?: TokenInfo | null;
```

Comment:
Don't do null and undefined together.

```suggestion
  senderToken?: TokenInfo;
```

In ToastController:
```
<TransactionToast
            onClose={() => toast.dismiss(t.id)}
            type={type}
            transaction={transaction}
            senderToken={senderToken || undefined}
            signerToken={signerToken || undefined}
            error={error}
          />
```